### PR TITLE
[BUGFIX2.0 Manual HOME - stepper always hold

### PR DESCRIPTION
**Pull request by android444**

### CODE REVIEW

File: Marlin/src/Marlin.cpp [hunk 1]
```diff
@@ -408,6 +408,7 @@ void manage_inactivity(bool ignore_stepper_queue/*=false*/) {
       if (!homeDebounceCount) {
         enqueue_and_echo_commands_P(PSTR("G28"));
         LCD_MESSAGEPGM(MSG_AUTO_HOME);
+		enqueue_and_echo_commands_P(PSTR("M84"));
```
Comment (thinkyhead) [inline Marlin/src/Marlin.cpp]: Tabs considered evil. See http://marlinfw.org/docs/development/coding_standards.html for guidelines on code formatting.

**Comment by thinkyhead** [comment]
What is the purpose of this change?

**Comment by android444** [comment]
After manual HOMING motors will not hold.

**Comment by GMagician** [comment]
@android444 I think this is a little bit dangerous. Disabling step motor will lead to an high probability to lose microstep. Not a good choice after home, don't you think?

**Comment by android444** [comment]
HOMING is made before printing.
Moving a few steps is irrelevant.

**Comment by thinkyhead** [comment]
> HOMING is made before printing.
Moving a few steps is irrelevant.

Sometimes homing is done in response to filament runout. Better not to lose any steps in such cases, even though I agree it's probably negligible.

This PR is not needed because steppers have a configured timeout already. After homing they may not disable right away, but if you walk away from the machine they will disable after a minute — or whatever is configured.

**Comment by android444** [comment]
Why?
This HOMING works only when you press a button.
What allows you to add this patch?

**Comment by thinkyhead** [comment]
Yes, I see it's a case where you have a HOME button or pin, so this is a handy way to issue a `G28`. But does that mean every user who has a HOME button wants this behavior? Again, there is a configurable timeout for steppers, so why is there a need to disable steppers for the HOME button only?

Your original description is not very detailed or explanatory.

**Comment by android444** [comment]
If HOME is with G-CODE, MOTOR must hold it.
If HOME is from a button, MOTOR does not have to hold, because later is HOME with G-CODE.
What other explanations do you want?

**Comment by thinkyhead** [comment]
You started out with **zero**, so even **one** is better, maestro. Please forgive my dullness.

**Comment by android444** [comment]
I dont't understand.

**Comment by thinkyhead** [comment]
What I'm saying is **this is a great idea**, but it should be an **option**.

**Comment by android444** [comment]
Which makes it impossible to work?

**Pull request closed without merging**